index.html

```diff
@@ -47,9 +47,41 @@ <h2>
     <section id="showcase">
       <div class="container">
         <h3>
-          Lorem ipsum dolor sit amet consectetur adipisicma enim ipsam. Minima
-          est eaque vitae itaque ex aspernatur sint at delectus.
+          Contactanos 
+          <p>Horario de lunes a viernes de 9am a 8pm y sabados de 9am a 1pm.</p>
+          <img src="./img/nicolaspugagdl.jpg" alt="Ubicacion">
```

Comment:
This should not be an image, it should be a map where the use can actually move inside google maps